src/libsyntax/diagnostic_list.rs

```diff
@@ -224,4 +291,5 @@ register_diagnostics! {
     E0555, // malformed feature attribute, expected #![feature(...)]
     E0556, // malformed feature, expected just one word
     E0557, // feature has been removed
+    E0584, // file for module `..` found at both .. and ..
```

Comment:
Don't know how to trigger this one and not enough time to investigate. Someone will do it later on if I don't myself.